hack/istio/install-istio-kiali-via-helm.sh

```diff
@@ -358,7 +358,7 @@ else
   ${HELM_EXE} template "${ISTIO_DIR}/install/kubernetes/helm/istio-init" --name istio-init --namespace ${NAMESPACE} | ${CLIENT_EXE} apply -f -
   _crd_count="0"
   echo -n "Waiting for the CRDs to be created"
-  while [ "$_crd_count" -lt "23" ]; do
+  while [ "$_crd_count" -lt "24" ]; do
```

Comment:
Is this really a good way of doing this? Because:

1. It changes over time and versions
2. If cert-manager is enabled it is different (like 28 instead)

Seems like a pretty fickle solution.

Reply:
its just our little hack script - so not end-user-facing... so at the end of the day "it works" is good enough.

It's also what the Istio docs say to do: https://istio.io/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template

```
Verify that all 23 Istio CRDs were committed to the Kubernetes api-server using the following command:

If cert-manager is enabled, then the CRD count will be 28 instead.

"$ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l"
```

Reply:
Fine then 🍅 🚽 👅

Reply:
@mtho11 translation please ;)